packages/node_modules/pouchdb-adapter-indexeddb/src/getLocal.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
```

Comment:
```suggestion
```

Please stop introducing `'use strict';` to new files.
It has no effect in ES modules (when using import/export):
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode?retiredLocale=de#strict_mode_for_modules

Reply:
:+1: Same for webpack?

Reply:
Short answer: Yes.

![PouchDB in strict](https://github.com/pouchdb/pouchdb/assets/1261013/ec4edef1-d4d7-4dda-917e-7217d67f2a97)

Details:
The whole bundle (everything in dir packages/) is in strict mode.
Some other packages (I see EventEmitter (polyfill), spark-md5, and vuvuzela in pouchdb-webpack) have there own configuration.

In the minified version, It's not that obvious.
Put this line in exports of webpack.config.js to get the unminified version:
```
optimization: { minimize: false },
```